test/chplcheck/BoolLitInCondStatement.chpl

```diff
@@ -0,0 +1,20 @@
+module BoolLits {
+  proc myProc () {
+    if(true) then {
```

Comment:
Not sure if this is intentional, but Chapel style is typically to omit `()` around if conditions. Also, if you have `{`, you shouldn't use `then`.

Reply:
Those are both candidates for linter rules -- though not in this PR, of course :)